install/kubernetes/helm/istio/values.yaml

```diff
@@ -342,6 +342,10 @@ global:
 
   # The trust domain corresponds to the trust root of a system
   # Refer to https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain
+  # Indicate the domain used in SPIFFE identity URL
+  # The default depends on the environment.
+  #   kubernetes: cluster.local
```

Comment:
Is this correct ? AFAIK on kubernetes it should match the k8s domain ( it can be customized in mesh config ).
Could you check, and if true reference the other setting ( and do we have a way to set it via values.yaml )?

In general - we would like consistency between the domain set on the cluster and identity.